runtime/onert/core/src/ir/OperationValidator.cc

```diff
@@ -375,9 +375,12 @@ void OperationValidator::visit(const operation::Pad &node)
   if (isPadV2)
   {
     const auto value_index{node.getInputs().at(operation::Pad::Input::VALUE)};
-    OP_REQUIRES(isSameType(input_index, value_index));
-    if (isQuantType)
-      OP_REQUIRES(isSameQuantParam(input_index, value_index));
+    const auto input_t = operandType(input_index);
+    const auto value_t = operandType(value_index);
+    const bool cond_f16 = (input_t == DataType::FLOAT16 && value_t == DataType::FLOAT16);
+    const bool cond_f32 = (input_t == DataType::FLOAT32 && value_t == DataType::FLOAT32);
+    const bool cond_q8a = (input_t == DataType::QUANT_UINT8_ASYMM && value_t == DataType::INT32);
```

Comment:
@wateret I think `QUANT_INT8_ASYMM` case is missing.

Reply:
@wateret On tensorflow lite, `QUANT_UINT8_ASYMM ` and `QUANT_INT8_ASYMM` requires same `input` and `value` data type.

Reply:
@hseok-oh Then ONERT Pad op should support both. Otherwise we may need to separate as two different Pad ops.

Reply:
>Then ONERT Pad op should support both.

Right. Other operation may consider both case. Please update to support both API.